CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing
+ 
+Thank you for considering contributing to Symfony UX! 
+
+Symfony UX is an open source, community-driven project, and we are happy to receive contributions from the community!
+
+> [!TIP]
+> It's a good idea to read the [Symfony's Contribution Guide](https://symfony.com/doc/current/contributing/index.html) first, even if not all of it applies to Symfony UX and should be adapted to this project (e.g.: Symfony UX has only one base branch, `2.x`).
+
+## Reporting an issue
+
+If you either find a bug, have a feature request, or need help/have a question, please [open an issue](https://github.com/symfony/ux/issues/new/choose).
+
+Please follow the issue template and provide as much information as possible,
+and remember to follow our [Code of Conduct](https://symfony.com/doc/current/contributing/code_of_conduct/index.html)
+as well, to ensure a friendly environment for all contributors.
+
+## Contributing to the code and documentation
+
+Thanks for your interest in contributing to Symfony UX! Here are some guidelines to help you get started.
+
+### Forking the repository
+
+To contribute to Symfony UX, you need to fork the repository on GitHub. This will give you a copy of the code under your GitHub user account.
+
+After forking the repository, you can clone it to your local machine:
+
+```shell
+$ git clone git@github.com:<USERNAME>/symfony-ux.git symfony-ux
+$ cd symfony-ux
+# Add the upstream repository, to keep your fork up-to-date
+$ git remote add upstream git@github.com:symfony/ux.git
+```
+
+### Setting up the development environment
+
+To set up the development environment, you need the following tools:
+
+- [PHP](https://www.php.net/downloads.php) 8.1 or higher
+- [Composer](https://getcomposer.org/download/)
+- [Node.js](https://nodejs.org/en/download/package-manager) 22 or higher
+- [Yarn](https://yarnpkg.com/) 4 or higher
+
+With these tools installed, you can install the project dependencies:
+
+```shell
+$ composer install
+$ yarn install
```

Comment:
I could not wait to see how you'd handle this :) To be honest 50% at least of the PR do not need JS, and in a following step i think we should make a distinction (same think for documentation).

But for now i like the simplicity of sharing guidelines!